api/core/tools/provider/builtin/sparkimagegeneration/tools/sparkimggen.yaml

```diff
@@ -0,0 +1,33 @@
+identity:
+  name: Sparktex2img
+  author: Onelevenvy
+  label:
+    en_US: Sparktex2img
+    zh_Hans: 讯飞星火图片生成
+    pt_BR: Sparktex2img
+  icon: icon.png
+  description:
+    en_US: Sparktex2img
+    zh_Hans: 讯飞星火图片生成
+    pt_BR: Sparktex2img
+  
+description:
+  human:
+    en_US: Sparktex2img is a text to image tool
+    zh_Hans: 根据用户输入的文本绘制精美的图片，是文本到图片的工具，由讯飞星火提供图片生成api
+    pt_BR: Sparktex2img is a text to image tool
+  llm: parktex2img is a tool used to generate images from text
+parameters:
+  - name: prompt
+    type: string
+    required: true
+    label:
+      en_US: Prompt
+      zh_Hans: 提示词
+      pt_BR: Prompt
+    human_description:
+      en_US: Image prompt
+      zh_Hans: 图像提示词
+      pt_BR: Image prompt
+    llm_description: Image prompt of Sparktex2img, you should describe the image you want to generate as a list of words as possible as detailed
+    form: llm
```

Comment:
Our ci requires a empty line at the end of a yaml file, refer to https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.new_line_at_end_of_file